### DISCUSSION TITLE
Support for WebDAV

### DISCUSSION THREAD

**Discussion by srfsh** [HemantKArya/BloomeeTunes#31] [Ideas]
Hi,

Due various reason, some of which is related to low space on phones, would it be okay if you could support WebDAV (or other network storage protocols such as FTP or S3) as an alternative way for the locally-owned and/or downloaded music?

Many thanks.

**Comment by HemantKArya** [comment]
Hi @srfsh thank you so much for you suggestion it seem an interesting idea to implt. an alternative WebDAV like solution for this app but I think currently we need to improve this app more and more so that it at least provides some basic features like downloading, scrobbling, etc, so yeah, I will leave this issue open for future so that if anyone want to suggest anything else related to this idea or want to work will be always welcomed here, I am open for all discussion.

I guess it should be in discussion tab so that people can share their ideas there.

**Comment by srfsh** [comment]
Sure, thing. I am glad you're even considering it. :)

The give you more information, it is relatively cheaper to get a cloud storage for media. And you can often connect this over FTP, S3, WebDAV. In fact, there is even this project called rclone (supports Android through apps like RoundSync) that allows you to basically do this for pretty much any cloud storage provider if they don't expose the network APIs you're looking for.

**Reply by HemantKArya** [reply]
Okay sure I will also look into that. 🥰
